plugins/modules/database/misc/odbc.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, John Westcott <john.westcott.iv@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
```

Comment:
```suggestion
```
No longer needed

Reply:
Thanks for point that out. I see its true for devel but the 2.9 docs still indicate its required. Do you know if that is going to be back ported to 2.9?

Reply:
In 2.9 it is more a policy requirement. It is not really used - except by the docs to HTML conversion of the 2.9 docsite, which doesn't apply here, and the values are shown in ansible-doc, but it doesn't mind if they aren't there. So leaving it away won't change Ansible 2.9 compatibility.